lib/parallel_tests/cli.rb

```diff
@@ -81,8 +81,9 @@ def run_tests_in_parallel(num_processes, options)
         end
 
         report_number_of_tests(groups) unless options[:quiet]
-        test_results = execute_in_parallel(groups, groups.size, options) do |group|
-          run_tests(group, groups.index(group), num_processes, options)
+        test_results = execute_in_parallel(groups, groups.size, options) do |group, index|
+          test_env_number = options[:first_is_1] ? index + 1 : index
```

Comment:
the first_is_1 logic was not here before, so it should get removed or be removed from some other place ?

Reply:
You're right, good catch. I didn't trace the test_env down far enough; I see it in `test/runner.rb` now. 

I've updated the PR to remove line 85 and just use index.